### PR TITLE
Docs/readmodel tester

### PR DESCRIPTION
**Pull request by EugeniyBurmistrov**

### CODE REVIEW

File: docs/test-readmodels.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+id: test-readmodels
+title: Test Read Models
+---
+
+## Testing Tools
+
+The **resolve-testing-tools** package contains utilities that allow you to test Read Models and Sagas. The provided tools reproduce a Read Model's full lifecycle in the testing environment.
+
+To write a test, call the `givenEvents` function. This function takes an array of events and gives access to a number of functions that you can chain together to describe the test case. The following functions are available:
```
Comment (dirk-pieterse) [inline docs/test-readmodels.md:10]: What do you mean when you say "gives access to functions"?

File: docs/test-readmodels.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+      expect(shoppingLists[0]).toMatchObject({
+        id: aggregateId,
+        name: 'Products'
+      })
+    })
+```
+<!-- prettier-ignore-end -->
+
+In this example, the `.all` function called in the end of the call chain is the `ShoppingLists` Read Model's resolver function. It returns a promise that resolves to the resolver's response object.
```
Comment (dirk-pieterse) [inline docs/test-readmodels.md:53]:
```suggestion
In this example, the `.all` function called at the end of the call chain is the `ShoppingLists` Read Model's resolver function. It returns a promise that resolves to the resolver's response object.
```

File: docs/test-readmodels.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    .properties({
+      [RESOLVE_SIDE_EFFECTS_START_TIMESTAMP]: Number.MAX_VALUE
+    })
+
+  expect(result).toMatchSnapshot()
+})
+```
+<!-- prettier-ignore-end -->
+
+The `saga` function returns a promise that resolves to an object containing the following fields:
```
Comment (dirk-pieterse) [inline docs/test-readmodels.md:87]:
```suggestion
The `saga` function returns a promise that resolves to an object that contains the following fields:
```

File: docs/test-readmodels.md (hunk 1 above)
Comment (EugeniyBurmistrov) [reply docs/test-readmodels.md:10]: It returns an object that contains these functions. The idea is that you can access these functions through the `givenEvents` function.

**Pull request merged**